Kuroba/app/src/main/java/com/github/k1rakishou/chan/ui/controller/LogsController.java

```diff
@@ -109,11 +109,11 @@ public static String loadLogs(int linesCount) {
 
         InputStream outputStream = process.getInputStream();
         //This filters our log output to just stuff we care about in-app (and if a crash happens, the uncaught handler gets it and this will still allow it through)
-        String filtered = "";
+        StringBuilder filtered = new StringBuilder();
         for (String line : IOUtils.readString(outputStream).split("\n")) {
-            if (line.contains(getApplicationLabel())) filtered = filtered.concat(line).concat("\n");
+            if (line.contains(getApplicationLabel())) filtered = new StringBuilder((filtered + line) + "\n");
```

Comment:
I think it's better to just use "append()" here instead of creating a new StringBuilder() on every iteration.